footer.inc

```diff
@@ -66,69 +66,88 @@
 				<h5
 					class="header col s12 white-text light text-shadow: 4px 4px 7px #FFFFFF;">
 					/ month</h5>
-					
-				<br><br><br>
+			</div>	
+				
 	
 	
 				<!-- Modal Trigger -->
-	<div style="padding: 20% 0% 0% 0%">
+	<div class="center-align" style="padding: 5% 0% 0% 0%">
+  
   <script>
 	
   $(document).ready(function(){
-	     $('.modal').modal({
```

Comment:
Was this intentional?  It looks like you removed all the style changes you made to your modal.

Reply:
didn't mean to do that. what's going on? was working fine last night.